### PR TITLE
feat: migrating java-bigqueryconnection snippet

### PR DESCRIPTION
**Pull request by suztomo**
Migrating the snippet from googleapis/java-bigqueryconnection https://github.com/googleapis/java-bigqueryconnection/tree/8e1577003242eb019fb07be51fc7b874613e9d3b/samples/snippets

These tests ran fine in https://source.cloud.google.com/results/invocations/bef3f8f1-575b-49e6-979c-cf69c91505f4/targets (May)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [x] `pom.xml` parent set to latest `shared-configuration`
- [x] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [x] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by suztomo** [comment]
The checks failed:

```
[ERROR] Failures:
[ERROR]   CreateAwsConnectionIT.<clinit>:41->requireEnvVar:51 Environment variable AWS_ACCOUNT_ID is required to perform these tests.
[ERROR]   CreateConnectionIT.<clinit>:42->requireEnvVar:54 Environment variable MY_SQL_DATABASE is required to perform these tests.
[ERROR]   DeleteConnectionIT.<clinit>:42->requireEnvVar:54 Environment variable MY_SQL_DATABASE is required to perform these tests.
[ERROR]   GetConnectionIT.<clinit>:42->requireEnvVar:54 Environment variable MY_SQL_DATABASE is required to perform these tests.
[ERROR]   ListConnectionsIT.<clinit>:42->requireEnvVar:54 Environment variable MY_SQL_DATABASE is required to perform these tests.
[ERROR]   ShareConnectionIT.<clinit>:42->requireEnvVar:54 Environment variable MY_SQL_DATABASE is required to perform these tests.
[ERROR]   UpdateConnectionIT.<clinit>:42->requireEnvVar:54 Environment variable MY_SQL_DATABASE is required to perform these tests.
```

**Comment by suztomo** [comment]
Tests passed (`SUCCESS: 1`) but did it really ran the tests?

```
******************** TESTING PROJECTS ********************
btlr run --max-cmd-duration=40m **/pom.xml --git-diff origin/main... . -- /tmpfs/src/github/java-docs-samples/.kokoro/tests/run_test_java.sh
Collecting directories that match pattern...291 collected.
Checking for changes with "git diff"... [0 of 291 complete].
Running command(s)... [0 of 1 complete].
#
# bigquery/bigqueryconnection/snippets
#

[0.001s][warning][os,container] Duplicate cpuset controllers detected. Picking /sys/fs/cgroup/cpuset, skipping /tmpfs/docker/overlay2/88846b78b0e1cfb3bc8abe0cd4b11ab75ef3bbb4f2cb132786d936aadcd64e23/merged/sys/fs/cgroup/cpuset.

 Testing completed.




#
# Summary
#

SUCCESS: 1, FAILURE: 0, SKIPPED: 0, ERROR: 0, 
bigquery/bigqueryconnection/snippets..................................[ SUCCESS]


[ID: 9181609] Command finished after 287 secs, exit value: 0
```

**Comment by suztomo** [comment]
Added "kokoro:run" label.

File: bigquery/bigqueryconnection/snippets/pom.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
```
Comment (Sita04) [inline bigquery/bigqueryconnection/snippets/pom.xml:4]:
```suggestion
  <groupId>com.example.bigquery</groupId>
```

**Review by Sita04 — APPROVED** [review]
LGTM. One minor change in pom.xml
Thanks Tomo for migrating these samples.

**Pull request merged**